include/hal/library/cryptlib/cryptlib_ec.h

```diff
@@ -45,6 +45,24 @@ extern void libspdm_ec_free(void *ec_context);
 extern bool libspdm_ec_set_pub_key(void *ec_context, const uint8_t *public_key,
                                    size_t public_key_size);
 
+/**
```

Comment:
If this function will not be called by any functions in `libspdm/library` it needs to be moved to `os_stub/spdm_crypt_ext_lib/cryptlib_ext.h`

Reply:
@steven-bellock , this is for cryptlib, not spdm_cryptlib.

Reply:
Ok.